content/learn/book/welcome/hello-bevy/_index.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "Hello, Bevy!"
+weight = 3
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+So, you've just made a project with Bevy. Great! However, currently, it isn't doing things 'the Bevy way'. Let's fix.
+
+## Bevy App Structure
+
+Bevy programs, which include games, tools and more, are conventially named 'Apps' and currently our Bevy program doesn't really follow the app structure or how bevy apps work.
+
+Below is a basic outline of how your Bevy Apps should look.
+
+```rust
+// MODULES (only the first line is required)
+use bevy::prelude::*;
+
+use another::module::*;
+
+mod some_file;
+mod another_file;
+
+// COMPONENTS AND RESOURCES
+
+struct Player;
+
+struct Health {
+    value: u32,
+}
+
+impl Health {
+    pub fn start() -> Self {
+       Self {
+           value: 100,
+       } 
+    }
+}
+// SYSTEMS
+
+fn setup(mut commands: Commands) {
+    commands.spawn()
+        .insert(Player)
+        .insert(Health::start());
+    println!("Setup Complete")
+}
+
+// MAIN
+
+fn main() {
+    App::build()
+        .add_system(setup.system())
+        .run();
+}
+```
+
+Bear in mind, the order of everything beyond modules is completely up to you.
+
+In detail:
+- First, you should import your modules. In order for a bevy program to work properly, it is **ESSENTIAL** that you import the bevy prelude, as this contains the App struct and other things required for bevy to work.
```

Comment:
This is a bit confusing: you can also manually import each needed element, and will definitely trip up advanced programmers.